### PR TITLE
Categoricals

### PR DESCRIPTION
**Pull request by mrocklin**
cc @lminer @jreback

### CODE REVIEW

**Comment by lminer** [comment]
huzzah!

File: partd/pandas.py [hunk 1]
```diff
@@ -104,16 +107,21 @@ def serialize(df):
     Uses Pandas blocks, snappy, and blosc to deconstruct an array into bytes
     """
     blocks, index, index_name, columns, placement = to_blocks(df)
+    categories = [b.categories if isinstance(b, pd.Categorical) else None
+                  for b in blocks]
+    blocks = [b.codes if isinstance(b, pd.Categorical) else b
+              for b in blocks]
```
Comment (jreback) [inline partd/pandas.py]: Categoricals can also be ordered (Boolean default False)

Comment (mrocklin) [inline partd/pandas.py]: Resolved.  FYI I ran into an issue that assert_frame_equal did not pick up on the ordered difference between two Categorical objects.

```python
In [1]: import pandas as pd

In [2]: pd.util.testing.assert_frame_equal(pd.DataFrame({'x': pd.Categorical([], ordered=True)}), pd.DataFrame({'x': pd.Categorical([], ordered=False)}))

In [3]: 
```

**Comment by lminer** [comment]
Are you planning on a new release soon to pull in this change or should I install the master branch?

**Comment by mrocklin** [comment]
I would like to issue a general dask release sometime late next week.
Perhaps install master branch until then?  If it's important I can also
issue a smaller release now.

**Comment by lminer** [comment]
Not important. Just checking. Thanks a bunch!

**Comment by mrocklin** [comment]
Released

**Pull request merged**